main/freetype/APKBUILD

```diff
@@ -2,15 +2,15 @@
 # Maintainer: Carlo Landmeter <clandmeter@gmail.com>
 pkgname=freetype
 pkgver=2.8
-pkgrel=0
+pkgrel=1
 pkgdesc="TrueType font rendering library"
-url="http://freetype.sourceforge.net"
+url="https://www.freetype.org/"
 arch="all"
 license="GPL"
 options="!check"
 depends=""
 depends_dev=""
-makedepends="$depends_dev zlib-dev libpng-dev bzip2-dev"
+makedepends="$depends_dev zlib-dev libpng-dev bzip2-dev harfbuzz-dev"
```

Comment:
I don't think we really need harfbuzz.
Because of the circular dep I will just remove it.